ait/_windows.py

```diff
@@ -307,7 +307,7 @@
 
 
 def _key_to_vk(key):
-    key = key.strip().upper()
+    key = key.strip(' ').upper()
```

Comment:
Makes me wonder if trying to press `' '` worked at all?

Reply:
good catch: press(' ') is still not working ...
Change key name to `'SPACE'` or remove `key.strip` completely?

Reply:
Perhaps we should special-case `' '`; I don't recall why I added the `.strip()` to be honest.